### DISCUSSION TITLE
Export map with assets.

### DISCUSSION THREAD

**Discussion by ItsTheLya** [4sval/FModel#464] [Q&A]
Hi, I'm currently exporting assets from a game Pacific Drive. I have a question however. Is there a way to export a .umap file with all the meshes inside it, as a part of the .umap file? Better yet, is there a way to export the .umap file as a .psk?

**Comment by t0stiman** [comment]
Did you ever find an answer to this question?

**Comment by interloko10** [comment]
If you want to load all the meshes of a umap file you can use BlenderUmap2 in Blender.
You can find it in this Patreon. The latest version is behind a pay wall but if you scroll down you can find some previous versions for free.
https://www.patreon.com/minshug

**Reply by t0stiman** [reply]
Nice, thanks for letting us know.
